model/minio/minio_server.rb

```diff
@@ -54,18 +63,20 @@ def init_health_monitor_session
     ssh_session.forward.local(UNIXServer.new(File.join(socket_path, "health_monitor_socket")), private_ipv4_address, 9000)
     {
       ssh_session: ssh_session,
-      minio_client: Minio::Client.new(
-        endpoint: connection_string,
-        access_key: cluster.admin_user,
-        secret_key: cluster.admin_password,
-        socket: File.join(socket_path, "health_monitor_socket")
-      )
+      socket_path: socket_path
     }
   end
 
   def check_pulse(session:, previous_pulse:)
     reading = begin
-      server_data = JSON.parse(session[:minio_client].admin_info.body)["servers"].find { _1["endpoint"] == endpoint }
+      admin_info = cluster.run(
```

Comment:
Using the `MinioCluster` model instead of the client to execute commands requires the same database to be a Minio cluster. This requirement makes it difficult to execute commands on external Minio clusters. I wish the `run` helper was part of the Minio client.

Reply:
@byucesoy I can make it a part of Client.initialize. Initially, I opt-in to this decision because I was planning to clean-up the certs file but that is not required since the certs can be made public and no harm would be done. If you think about it, if we had to keep the file created and deleted before and after every api call, there is simply no other way than introducing such an abstraction. 
@enescakir to be able to support what you want, the client needs to get the certificates in an argument in the initialize, would that be OK?

Reply:
I think it's fair. The client's constructor could possibly include an additional parameter, and if provided, it writes it to the file.

Reply:
updated